demos/gpu/README.md

```diff
@@ -44,7 +44,7 @@ The following prerequisites must be met to successfully run the Horovod demo tut
 
 The following prerequisites must be met to successfully run the RAPIDS demo tutorials (**demos/rapids**):
 
-- Your environment has one or more GPUs with the [NVIDIA Pascal](https://www.nvidia.com/en-us/geforce/products/10series/architecture/) architecture or better and [compute capability](https://developer.nvidia.com/cuda-gpus) 6.0+.
+- Your environment has one or more GPUs with the [NVIDIA Pascal](https://www.nvidia.com/en-us/geforce/products/10series/architecture/) architecture or better and [CUDA](https://developer.nvidia.com/cuda-gpus) 6.0+.
```

Comment:
The phrasing in the current doc was copied from the prerequisites on the [RAPIDS getting-started page](https://rapids.ai/start.html); see the **[Prerequisites](https://rapids.ai/start.html#prerequisites)** section; (when using the direct link, you might need to scroll up a bit to see the section title and the "GPU" line):
> **GPU:** NVIDIA Pascal™ or better with **[compute capability](https://developer.nvidia.com/cuda-gpus)** 6.0+
> ...
> **CUDA: [9.2](https://developer.nvidia.com/cuda-92-download-archive) with driver v396.37+** or **[10.0](https://developer.nvidia.com/cuda-10.0-download-archive) with driver v410.48+**

=> I prefer to stick with RAPIDS's phrasing.

Reply:
Accepted